apps/emqx_management/test/emqx_mgmt_api_SUITE.erl

```diff
@@ -322,7 +322,7 @@ t_plugins(_) ->
                                  auth_header_()),
     [Plugin3] = filter(get(<<"data">>, Plugins3), <<"name">>, <<"emqx_auth_mnesia">>),
     ?assertEqual(<<"emqx_auth_mnesia">>, maps:get(<<"name">>, Plugin3)),
-    ?assertEqual(false, maps:get(<<"active">>, Plugin3)),
+    ?assertEqual(true, maps:get(<<"active">>, Plugin3)),
```

Comment:
I have no idea why this test case was passing before .

Reply:
it looks very much like a copy-paste from line 311.
line 311: put `unload` then expect "active" to be `false`, which makes sense.
line 325: put `load` then expect "active" to be `false` still ? which looks like a bug in the code then a bug in the test to make it pass.